<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceWidget</class>
 <widget class="QWidget" name="PreferenceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>133</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lb_algorithm">
       <property name="text">
        <string>Algorithm</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lb_edgeLength">
       <property name="text">
        <string>EdgeLength</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_edgeLength"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lb_iteration">
       <property name="text">
        <string>Iteration</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cb_iteration"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lb_tryGoal">
       <property name="text">
        <string>TryGoal</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cb_tryGoal"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_algorithm"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>rrtTextChanged(QString)</signal>
 </slots>
</ui>
